tests/unit/plugins/inventory/test_linode.py

```diff
@@ -39,10 +40,11 @@ def inventory():
     return InventoryModule()
 
 
-def test_access_token_lookup(inventory):
+def test_empty_access_token_lookup(inventory):
```

Comment:
```suggestion
def test_missing_access_token_lookup(inventory):
```
For "empty" I would expect an empty string :)

Reply:
Ah, yeah. Fair enough. Committed!

@felixfontein Let me know if you need me to squash. Thanks again for your 👀 and feedback.